apps/condo/k6/src/registerBillingReceiptFile.test.ts

```diff
@@ -0,0 +1,184 @@
+            exec: 'registerBillingReceiptFileCase',
+            executor: 'constant-vus',
+            duration: DURATION,
+            vus: 1,
+        },
+    },
+    thresholds: {
+        http_req_failed: ['rate<0.01'],
+        http_req_duration: ['med<14000'],
+    },
+}
+
+export function setup () {
+    const { token } = setupCondoAuth(true)
+
+    const createdOrganization = createOrganization({ token })
+    const organizationId = createdOrganization.json('data.obj.id')
+    const billingIntegration = createBillingIntegration({ token })
+
+    const billingContext = createBillingIntegrationOrganizationContext(
+        { token },
+        { id: organizationId },
+        billingIntegration.json('data.obj'),
+        { status: 'Finished' },
+    )
+    const data = {
+        token,
+        organizationId,
+        billingContext: billingContext.json('data.obj'),
+    }
+
+    const { receipts } = registerBillingReceipts(data)
+
+    const fileSize = 1024 * 1024 * 10 // 1024 bytes (kb) * 1024 => mb * 10 => 10mb
+    const bytes = crypto.randomBytes(fileSize)
+    const base64EncodedPDF = encoding.b64encode(bytes)
+
+    return {
+        ...data,
+        receipts,
+        base64EncodedPDF,
+    }
+}
+
+const randomNumber = (numDigits) => {
+    const min = 10 ** (numDigits - 1)
+    const max = 10 ** numDigits - 1
+    return faker.datatype.number({ min, max })
+}
+
+const createAddressWithUnit = () => __ENV.BASE_URL === 'http://localhost:3000'
+    ? `${faker.address.cityName()} ${faker.address.streetAddress(true)}`
+    : 'г Нижний Новгород, пр-кт Ленина, д 88 к 78, кв 1'
+
+const generateServicesData = (count = 3, toPay = '') => {
+    const services = []
+
+    for (let i = 0; i < count; i++) {
+        services.push({
+            id: faker.datatype.number().toString(),
+            name: faker.random.alphaNumeric(),
+            toPay: toPay !== '' ? toPay : faker.datatype.number().toString(),
+            toPayDetails: {
+                formula: 'charge + penalty',
+                charge: faker.datatype.number().toString(),
+                penalty: faker.datatype.number().toString(),
+            },
+        })
+    }
+    return services
+}
+
+const createRecipient = (extra = {}) => ({
+    tin: '3703048756',
+    routingNumber: '046577674',
+    ...extra,
+})
+
+const createJSONReceipt = (extra = {}) => {
+    const [month, year] = dayjs().add(-1, 'month').format('MM-YYYY').split('-').map(Number)
+    return Object.fromEntries(Object.entries({
+        importId: faker.datatype.uuid(),
+        address: createAddressWithUnit(),
+        accountNumber: randomNumber(10).toString(),
+        toPay: faker.finance.amount(-100, 5000),
+        month, year,
+        services: generateServicesData(faker.datatype.number({ min: 3, max: 5 })),
+        ...createRecipient(),
+        raw: extra,
+        ...extra,
+    }).filter(([, value]) => !!value))
+}
+
+const registerBillingReceipts = (data) => {
```

Comment:
You have same utils in `registerBillingReceipts.test.ts`. Mb move it to utils?

Reply:
I'll move it once implement registerBillingReceipts.test.ts